Marlin/src/inc/Conditionals_post.h

```diff
@@ -1619,7 +1619,9 @@
 // If platform requires early initialization of watchdog to properly boot
 #define EARLY_WATCHDOG (ENABLED(USE_WATCHDOG) && defined(ARDUINO_ARCH_SAM))
 
-#define USE_EXECUTE_COMMANDS_IMMEDIATE ENABLED(G29_RETRY_AND_RECOVER)
+#ifndef USE_EXECUTE_COMMANDS_IMMEDIATE
+  #define USE_EXECUTE_COMMANDS_IMMEDIATE ENABLED(G29_RETRY_AND_RECOVER)
+#endif
```

Comment:
This isn't quite compatible with the usage of `USE_EXECUTE_COMMANDS_IMMEDIATE`. Configurable switches are simply defined with `#define THE_SWITCH` and can be tested with `#if ENABLED(THE_SWITCH)`, whereas conditionals are set to a pre-compiler true/false value and in the code are tested with `#if THE_CONDITION`. So this would require someone to add a line like `#define USE_EXECUTE_COMMANDS_IMMEDIATE 1` to their configuration. But doing so wouldn't be of any benefit because the presence of the immediate sub-commands functions is entirely dependent on features that use it being enabled, so enabling them outside of having any features that use them is only useful for developer hacking.

Reply:
> But doing so wouldn't be of any benefit because the presence of the immediate sub-commands functions is entirely dependent on features that use it being enabled, so enabling them outside of having any features that use them is only useful for developer hacking.

You are looking at it just from the perspective of individual users, for companies like ours, there has always been a need to tweak the firmware beyond what makes sense to push upstream. For example, I have modified our firmware to inject additional commands after a print abort. I think "process_subcommands_now_P" is an effective way to make small tweaks like that, far superior than having to write a bunch of C++ code.